server/docs/api.md

```diff
@@ -28,6 +28,15 @@ $ curl -H "Accept: application/json" https://bookworms-api.vercel.app/api/ping
 
 ### GET /api/reviews
 
+
+This api does three things
+
+- Get all reviews (By default are arranged in descending order of the date they were created meaning the most recent comes first)
+- Get all reviews with a partular keyword in review body(text) 
+- Get reviews  ordered by likes or books names
```

Comment:
In our reqruiement, we also need to sort by recency: https://github.com/codeday-labs/bookworms/projects/1#card-64509875

By recency means user can to sort reviews by its `created_at` field.

Reply:
Books names is a bonus, but I am glad you did that alr. :smile: